README.md

```diff
@@ -58,6 +68,232 @@ dependencies {
+    container: ViewGroup?,
+    savedInstanceState: Bundle?
+): View = paymentFormMediator.provideFragmentContent(this)
+```
+
+## Make it your own
+<sub>!!! Any customisation needs to be done before creating Payment form mediator. Once mediator is created, any changes done on style will not be reflected in the UI !!!</sub>
+
+We have been working hard to provide a selection of ways to make it your own. In order of complexity we'll start with:
+
+### Modify Default
+In our [Get started](#Get-started) example we have used Default Style to get something working quickly. If that was mostly what you were looking for, then modify individual properties should be the best option for you.
+
+```kotlin
+val paymentFormStyle = PaymentFormStyle()
+val customInputComponentStyle = DefaultLightStyle.inputComponentStyle(
+    titleTextId = R.string.custom_card_number_title,
+    withLeadingIcon = true,
+    margin = Margin(bottom = 16)
+)
+paymentFormStyle.paymentDetailsStyle.cardNumberStyle.inputStyle = customInputComponentStyle
+```
+
+### Use Theme
+Detailed implementation of this approach can be found in `CustomPaymentFormTheme.kt`. With the Theme, we are aiming to give you a design system that you can use to create the full UI style by providing a small number of properties that we will share across to sub components. Since you might not fully agree with our mapping, you can still individually change each component afterwards (as in the Modify Default example).
+
+```kotlin
+val theme = PaymentFormTheme(
+    paymentFormThemeColors = paymentFormThemeColors,
+    /**
+     * option 1: Use combination of default and custom components
+     */
+    paymentFormComponents = DefaultPaymentFormTheme.providePaymentFormComponents(
+        cardNumber = cardNumber,
+        addressLineOne = addressLineOne,
+        addressLineTwo = addressLineTwo,
+        addBillingSummaryButton = addBillingSummaryButton,
+        editBillingSummaryButton = editBillingSummaryButton
+    ),
+    /**
+     * option 2: Use default components
+     * paymentFormComponents = DefaultPaymentFormTheme.provideComponents()
+     */
+    paymentFormShape = PaymentFormShape(
+        inputFieldShape = Shape.RoundCorner,
+        addressSummaryShape = Shape.Rectangle, buttonShape = Shape.Circle
+    ),
+    paymentFormCornerRadius = PaymentFormCornerRadius(
+        inputFieldCornerRadius = CornerRadius(INPUT_FIELD_CORNER_RADIUS),
+        addressSummaryCornerRadius = CornerRadius(INPUT_FIELD_CORNER_RADIUS)
+    )
+)
+val style = PaymentFormStyleProvider.provide(theme)
+```
+We think this approach should hit a good balance between great control of UI & simple, concise code. However if you still find the mapping to need excessive customisation, our final approach may be more to your liking.
+
+### Declare all components
+This is by no means the easy way, but it is absolutely the way to fully customise every property, and discover the full extent of customisability as you navigate through.
```

Comment:
Can we add a support doc for a fully custom version sample version by providing a reference for CustomPaymentDetailsStyle.kt class?

Reply:
Make sense, as it was just created.

Reply:
Updated.